sass/tokens/_index.scss

```diff
@@ -65,18 +65,11 @@
 @forward 'vars';
 
 /// ## Import
-/// If you already imported `accoutrement/sass/tools` you are ready to go.
+/// If you've already imported `accoutrement/sass/tools` you are ready to go.
```

Comment:
(see earlier comment about requiring `/sass/tools`)

Reply:
removed all of these